WeakAurasOptions/BuffTrigger2.lua

```diff
@@ -144,7 +144,21 @@ local function GetBuffTriggerOptions(data, optionTriggerChoices)
           and not IsSingleMissing(trigger)
           and trigger.use_debuffClass)
       end,
-      values = WeakAuras.debuff_class_types
+      values = WeakAuras.debuff_class_types,
+      get = function(info, index)
```

Comment:
Did you test this with multi-selected auras?

Reply:
i didn't, and it doesn't work indeed

Reply:
I was afraid of that. I wish we didn’t have to handle multi select, it’s so annoying :(

Reply:
correction, it was working, i miss tested because it's very rare i do multi-selection changes with different state on each aura and didn't know how it was showing
https://i.imgur.com/uOT6Cg4.gif

Reply:
I still don’t really like that. Every other control in WeakAuras (aside from custom code) obeys the requirement that after one user interaction, any discrepancy is eliminated. Also, the name is always blue if there is a discrepancy. The behavior of this control does not conform to this requirement, which makes it unique amongst non-code controls.

Think about what the user would assume if they saw that. They’d assume the same thing you did, and ask why this doesn’t work.

Reply:
fixed